js-miniapp-sample/src/pages/user-details.js

```diff
@@ -310,11 +308,31 @@ function UserDetails(props: UserDetailsProps) {
           )}
           {props.contactList &&
             props.contactList.map((contact) => (
-              <ListItem>
+              <ListItem divider>
                 <ListItemAvatar>
                   <Avatar className={classes.contactIcon} />
                 </ListItemAvatar>
-                <ListItemText primary={contact.id} />
+                <ListItemText
+                  primary={contact.id}
+                  secondary={
+                    <React.Fragment>
+                      <Typography>
+                        {contact.name !== '' &&
+                        contact.name !== undefined &&
+                        contact.name !== null ? (
+                          <span>{'Name: ' + contact.name}</span>
+                        ) : null}
```

Comment:
I think you don't need to manually check `undefined` and `null`. React treats the value `false` the same as `null` - it displays nothing.

```suggestion
                        {contact.name && contact.name !== '' && (
                          <span>{'Name: ' + contact.name}</span>
                        )}
```